source/functions/BEMIO/writeH5/h5bemWriteBody.m

```diff
@@ -0,0 +1,34 @@
+function h5bemWriteBody(filename, bodyNumber, cb, cg, dispVol, K, re, im, addedMass, addedMassInf, radiationDamping)
+	% Function to write a body's properties and coefficients to an 
+    %     existing WEC-Sim/BEMIO-formatted h5 file
```

Comment:
I think this needs to be removed, and has been replaced by ``source/functions/BEMIO/writeH5.m``